composer.json

```diff
@@ -9,8 +9,10 @@
     },
     "require-dev": {
         "brain/monkey": "^2",
+        "dealerdirect/phpcodesniffer-composer-installer": "0.5.0",
```

Comment:
Without this, sometimes there are error messages when doing `npm run lint:php`, like:

>ERROR: Referenced sniff "WordPress" does not exist